Userland/Libraries/LibVideo/PlaybackManager.h

```diff
@@ -153,12 +165,13 @@ class PlaybackManager {
 
     static DecoderErrorOr<NonnullOwnPtr<PlaybackManager>> create(NonnullOwnPtr<Demuxer> demuxer, PlaybackTimerCreator playback_timer_creator);
 
-    void start_timer(int milliseconds);
     void timer_callback();
+    // This MUST be called with m_demuxer_mutex LOCKED!
```

Comment:
Instead of adding a comment here, maybe you can check this invariant in the code?

Reply:
I wanted to, but Mutex has no way to verify a lock. I don't know whether there's a good way to add that as well, but if there is feel free to let me know.

Reply:
It seems trivial to add something based on the value of `Mutex::m_lock_count`, but exposing it could create new race conditions where we'd rather have none, so let's not do that. Let's leave it as it is!